suite-native/formatters/src/components/FiatBalanceFormatter.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { Box, Text } from '@suite-native/atoms';
+import { useFormatters } from '@suite-common/formatters';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { FormatterProps } from '../types';
+import { EmptyAmountText } from './EmptyAmountText';
+import { AmountText } from './AmountText';
+
+/** Matches three groups: 1. currency symbol,  2. whole number part and 3. decimal part. */
+const BALANCE_REGEX = /^(\D+)([\d,]+)(?:\.(\d+))?$/u;
+
+type BalanceFormatterProps = FormatterProps<string | null>;
+
+const wholeNumberStyle = prepareNativeStyle(utils => ({
+    flexShrink: 1,
+    marginBottom: -utils.spacings.small,
+    textAlign: 'center',
+}));
+
+const parseBalanceAmount = (value: string) => {
+    const regexGroups = value.match(BALANCE_REGEX);
+    const [_, currencySymbol, wholeNumberPart, decimalNumberPart] = regexGroups ?? [
+        null,
+        null,
+        null,
+    ];
+
+    return {
+        currencySymbol,
+        wholeNumber: wholeNumberPart,
+        decimalNumber: decimalNumberPart ? `.${decimalNumberPart}` : '',
+    };
+};
+
+export const FiatBalanceFormatter = ({ value }: BalanceFormatterProps) => {
```

Comment:
I agree this formatter is very specific in terms of styling. But since the FiatBalanceFormatter is not only used in the graph headers but also in the `AccountImportSummaryScreen`, I've decided to put it in the `@suite-native/formatters` package. I believe this could be reused later also in other screens and packages.